external/llvm-project/mlir/lib/Dialect/AMDGPU/IR/AMDGPUDialect.cpp

```diff
@@ -185,6 +185,34 @@ void RawBufferAtomicUminOp::getCanonicalizationPatterns(
   results.add<RemoveStaticallyOobBufferWrites<RawBufferAtomicUminOp>>(context);
 }
 
+//===----------------------------------------------------------------------===//
+// WMMAOp
+//===----------------------------------------------------------------------===//
+LogicalResult WMMAOp::verify() {
```

Comment:
While you're here, verify attribute sanity (especially with the unit attribute changes above). Make sure, for example, that `unsignedA` is only present on integer operations, or that you're not trying to set a `subwordOffset` that's too large for the given type.